layouts/partials/events/render-event.html

```diff
@@ -72,11 +74,10 @@ <h2>Examples</h2>
     */}}
     {{ if $desired_example_name }}
         {{ if eq $example_name $desired_example_name }}
-            {{ $example_content := partial "json-schema/resolve-refs" (dict "schema" $example "path" "event-schemas/examples") }}
-```json
-{{ jsonify (dict "indent" "  ") $example_content }}
-```
+            {{ partial "events/example" (dict "example" $example)}}
```

Comment:
given this only takes a single parameter, it might be easier to pass it directly, rather than wrap it in a dict?

```suggestion
            {{ partial "events/example" $example }}
```